site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+This is summary of Wednesday 11th August's Data Ethics Club discussion, where we spoke and wrote about the Fast Company article [What an ancient lake in Nevada reveals about the future of tech](https://www.fastcompany.com/90618225/what-an-ancient-lake-in-nevada-reveals-about-the-future-of-tech) by Kate Crawford.
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+```
+## Introduction 
+
+In this Data Ethic's club meeting we read a piece by Kate Crawford: [What an ancient lake in Nevada reveals about the future of tech](https://www.fastcompany.com/90618225/what-an-ancient-lake-in-nevada-reveals-about-the-future-of-tech). 
+
+The author recalls her drive through a desert valley in Nevada, observing the mining boon in the area. The area is a hotbed for Lithium mining, a material that is a key in the manufacture of rechargeable batteries. Whilst such batteries were never designed for being used to power machines as large as a car (with Tesla being the world's leading consumer of lithium-ion batteries), the batteries remain the best option. 
+
+Whilst electric cars are cited as an excellent solution to our ever increasing carbon emissions. However, as the article notes: "the mining, smelting, export, assemblage, and transport of the battery supply chain has a significant negative impact on the environment and, in turn, on the communities affected by its degradation."
+
+This aside, most electricity used to charge these lithium car batteries is taken from the national grid. In the US, less than a fifth of this electricity comes from renewable energy sources.
+
+This brings us to a broader issue with new technology, where subtle linguistics makes us believe that developments are far less physical than they sound. 
+
+The idea of storing data in the "cloud" immediately brings images of an almost ethereal, mushy collection of data in the sky. Indeed, we *up*load data to the cloud, implying something about the trajectory of the data in question. This conventionally sidesteps the much less sheep-shaped reality that the cloud is just a collection of servers somewhere. Indeed the internet itself, despite being touted as "wireless" is anything but (see this cool little [diagram](https://www.submarinecablemap.com/) and this breakdown of [how the internet works](https://tutorial.djangogirls.org/en/how_the_internet_works/)). All of these physical systems involving servers, smartphones and laptops have lithium batteries at their core.
+
+For every second of virtual time, we extract from Earth's geological history, building smartphones and other devices that have lifetimes of a few years, often [artificially reduced](https://www.theguardian.com/technology/2017/dec/29/apple-apologises-for-slowing-older-iphones-battery-performance). Is this practice really sustainable?
+
```

Comment:
I would just add a round up sentence here at the end of the intro like: 
"In our discussion of Crawford's article during this Data Ethics Club we touched on what it might take for the impact of tech on the environment to change, who would need to act for those changes to be effective, and considered what actions we could take as individuals."